src/schema/v2/homeView/__tests__/HomeView.test.ts

```diff
@@ -1,5 +1,6 @@
 import gql from "lib/gql"
 import { runQuery } from "schema/v2/test/utils"
+import { ResolverContext } from "types/graphql"
 
 describe("homeView", () => {
```

Comment:
The changes to this test file got verbose, but basically this is the state of things now:

<pre>
  <b>homeView</b>

    <b>sectionsConnection</b>
      with an unauthenticated user
        ✅ returns the correct sections
      with an authenticated user
        ✅ returns the correct sections

    <b>section</b>
      with a section that requires authentication
        with an unauthenticated user
          ✅ throws an error
        with an authenticated user
          ✅ returns the requested section
      with a section that does not require authentication
        with an unauthenticated user
          ✅ returns the requested section
        with an authenticated user
          ✅ returns the requested section
<pre>